index.d.ts

```diff
@@ -865,6 +874,10 @@ declare module 'plivo-browser-sdk/media/audioDevice' {
         * Check the input & output audio device for windows machine such that both belong to same groupid
         */
     export const setAudioDeviceForForWindows: (devices: any, lastConnectedMicDevice: any, lastConnectedSpeakerDevice: any) => void;
+    /**
+        * Check audio devices for electron app
+        */
+    export const checkElectronAudioDevices: () => void;
```

Comment:
Is it specific to Electron or can be used for non Electron environment (Firefox, Chrome, ...etc)?
If it can be used for non Electron environments, I would suggest changing the name from checkElectronAudioDevices to checkAudioDevices

Reply:
It is specific to electron only.